library/core/src/prelude/common.rs

```diff
@@ -41,6 +41,67 @@ pub use crate::option::Option::{self, None, Some};
 #[doc(no_inline)]
 pub use crate::result::Result::{self, Err, Ok};
 
+#[stable(feature = "core_prelude_extra", since = "CURRENT_RUSTC_VERSION")]
+#[doc(no_inline)]
+pub use crate::cell::{Cell, LazyCell, OnceCell, RefCell, SyncUnsafeCell, UnsafeCell};
```

Comment:
hmm, the OnceCell case seems like it may risk confusion due to `once_cell`.

Reply:
Hm, I *think* that the parts of `once_cell` that overlap with std are considered deprecated in favour of std (MSRV willing).

Reply:
I mean the author basically contributed the code, so yeah.